target/linux/ramips/dts/HC5962.dts

```diff
@@ -0,0 +1,156 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "HiWiFi HC5962";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x10000000>;
+	};
+
+	palmbus@1E000000 {
+		spi@b00 {
+			status="disabled";
+		};
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+	
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		status-white {
+			label = "hc5962:white:status";
+			gpios = <&gpio0 6 0>;
+		};	
+		
+		system-red {
+			label = "hc5962:red:system";
+			gpios = <&gpio0 7 0>;
+		};
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 18 1>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+	
+&nand {
+	status = "okay";
+
+	partition@0 {
+		label = "u-boot";
+		reg = <0x0 0x80000>;
+		read-only;
+	};
+
+	partition@80000 {
+		label = "debug";
```

Comment:
Are you sure it's safe (or maybe required?) to keep this partition writable? What kind of data are stored there?